easybuild/easyconfigs/n/NVHPC/NVHPC-23.1-CUDA-12.0.0.eb

```diff
@@ -0,0 +1,76 @@
+name = 'NVHPC'
+version = '23.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+local_tarball_tmpl = 'nvhpc_2023_%%(version_major)s%%(version_minor)s_Linux_%s_cuda_multi.tar.gz'
+# By downloading, you accept the HPC SDK Software License Agreement
+# https://docs.nvidia.com/hpc-sdk/eula/index.html
+# accept_eula = True
+source_urls = ['https://developer.download.nvidia.com/hpc-sdk/%(version)s/']
+sources = [local_tarball_tmpl % '%(arch)s']
+checksums = [
+    {
+        local_tarball_tmpl % 'aarch64':
+            '5b430e03752954ea62ac1c745b1735cfdaa43b2e981a9412c1465ecb0412fff6',
+        local_tarball_tmpl % 'ppc64le':
+            '81759e7c747bf4f552b75e7657301f76ecc0828b94fe860f81108c6e83e6ad2b',
+        local_tarball_tmpl % 'x86_64':
+            '55a064415f6d4ce6a01823ee27ebd266f4fb579679871e7c1a7c054bdc18e9f5',
+    }
+]
+
+local_gccver = '12.2.0'
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.39', '', ('GCCcore', local_gccver)),
+    # This is necessary to avoid cases where just libnuma.so.1 is present in the system and -lnuma fails
+    ('numactl', '2.0.16', '', ('GCCcore', local_gccver)),
+    ('CUDA', '12.0.0', '', SYSTEM),
```

Comment:
I am not sure if we prefer 11.8.0 or 12.0.0 for the 2022b toolchain generation.

Reply:
As this has a versionsuffix I am fine with this. We can always make the equivalent with CUDA 11.8 as well if we fix on that for 2022b.